### PR TITLE
Inline streaming of compatible audio and video

### PR DESCRIPTION
**Pull request by jorntx**
First stab at inline playback. Took me a lot of time to try to iron out some of the issues and ended up using some "hacks" that I'm not content with but I think it's good enough for a first run.

### CODE REVIEW

File: src/js/src/components/inlinePlayer/InlinePlayer.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    data() {
+        return {
+            playerInstance: null
+        }
+    },
+
+    mounted() {
+        this.playerInstance = videojs(this.$refs.inlinePlayerObj, this.getPlayerOptions())
+        this.playerInstance.on('error', () => {
+        const errorObj = this.player.error()
```
Comment (jesperlauridsen) [inline src/js/src/components/inlinePlayer/InlinePlayer.vue:37]: I get an error here saying 

IDEOJS: ERROR: TypeError: Cannot read property 'error' of undefined
    at Player.eval (InlinePlayer.vue?aa9c:37)
    at HTMLDivElement.data.dispatcher (video.es.js?f0e2:1899)
    at trigger (video.es.js?f0e2:2035)
    at Player.trigger$1 (video.es.js?f0e2:2921)
    at Player.error (video.es.js?f0e2:24603)
    at Player.eval (video.es.js?f0e2:24035)
    at eval (video.es.js?f0e2:4626)

Comment (jesperlauridsen) [reply src/js/src/components/inlinePlayer/InlinePlayer.vue:37]: To clarify, this happens when I try to play a video of the .wmv format.

Comment (jesperlauridsen) [reply src/js/src/components/inlinePlayer/InlinePlayer.vue:37]: Even more clarification: I believe i should get an error in our notification system - that doesn't seem to happen.

Comment (jorntx) [reply src/js/src/components/inlinePlayer/InlinePlayer.vue:37]: That's a nasty one....will look into this.

Comment (jorntx) [reply src/js/src/components/inlinePlayer/InlinePlayer.vue]: Found it! And good catch btw 👍 

It was a refactoring mishap on my part.

**Review by jesperlauridsen — APPROVED** [review]
Seems to work nicely now, good work!. Still a little underwhelmed by the native browser support different formats. But that's none of your fault. Approved \o/

**Pull request merged**